samples/sample-04/src/main/kotlin/com/remusrd/notesample/configuration/KafkaConfig.kt

```diff
@@ -0,0 +1,57 @@
+package com.remusrd.notesample.configuration
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.autoconfigure.kafka.KafkaProperties
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.http.converter.json.Jackson2ObjectMapperBuilder
+import org.springframework.kafka.core.*
+import org.springframework.kafka.support.serializer.JsonDeserializer
+import org.springframework.kafka.support.serializer.JsonSerializer
+
+@Configuration
+class KafkaConfig {
+    @Autowired
+    lateinit var jackson2ObjectMapperBuilder: Jackson2ObjectMapperBuilder
+    @Autowired
+    lateinit var kafkaProperties: KafkaProperties
+
+
+    @Bean
+    fun kafkaTemplate(): KafkaTemplate<Any, Any> {
```

Comment:
If we rely on Spring Boot, I don't see reason to declare this bean.

Reply:
This was because IntelliJ says there are no beans of type KafkaTemplate, I guess it is a bug of some type